server/index.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.listen(process.env.PORT || 3000, () => {
+  console.log('server is running on port 3000');
```

Comment:
Can you initiate a database connection here using the function created by @brianineza01 ?